iroha_client/Cargo.toml

```diff
@@ -17,6 +17,7 @@ is-it-maintained-open-issues = { repository = "https://github.com/hyperledger/ir
 maintenance = { status = "actively-developed" }
 
 [dependencies]
+iroha = { path = "../iroha" }
```

Comment:
This should not be the case, the things that we need from `iroha` need to be moved into data model.

Reply:
Ok, i guess it can be separated into 2 traits (one for output type and one for actual query execution).